src/GraphQL/Attributes/Authorization/AuthorizeAttribute.cs

```diff
@@ -29,13 +29,29 @@ public AuthorizeAttribute(string policy)
     /// <summary>
     /// The name of policy to apply.
     /// </summary>
-    public string? Policy { get; init; }
+    public string? Policy
+    {
+        get;
+#if NET5_0_OR_GREATER
+        init;
+#else
+        set;
+#endif
```

Comment:
Minimize impact here.  Perhaps just converting to `set;` would be better.

See https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/init#breaking-changes

Reply:
Note that this class is not likely to be inherited in 3rd-party libraries, so non-source-breaking is probably acceptable here.

Reply:
Changed it to simply `set;` since we wouldn't want a class library compiled against .NET Standard 2.0 to assume that the property was an `init;` while the main project is compiled against .NET 6 and assumes the property was a `set;` therefor having a broken build without any 3rd party references.